src/cli/mod.rs

```diff
@@ -1,9 +1,6 @@
 pub(crate) mod cmd;
 mod instrumentation;
 
-// For use in console output
-pub(super) const FLAKEHUB_WEB_ROOT: &str = "https://flakehub.com";
```

Comment:
Just a minor note that I think now all the URLs will be `api.flakehub.com`, I don't know if you care about that... (because it uses `self.api_addr`, which is `api.flakehub.com`)